src/Navigator/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { createBottomTabNavigator } from 'react-navigation-tabs';
+import TabBarIcon from './TabBarIcon';
+import Stats from '../Stats/index';
+import TransCreator from '../TransCreator/index';
+import Personal from '../Personal/index';
+import themeColor from '../Common/Color';
+
+const AppNavigator = createBottomTabNavigator({
+  Stats,
+  Create: TransCreator,
+  Me: Personal,
+},
+{
+  defaultNavigationOptions: ({ navigation }) => ({
+    // eslint-disable-next-line react/prop-types
+    tabBarIcon: ({ focused, tintColor }) => (
+      <TabBarIcon
+        navigation={navigation}
+        focused={focused}
+        tintColor={tintColor}
+      />
+    ),
+
+  }),
+  tabBarOptions: themeColor,
```

Comment:
It is not a good idea to map to whole color export to the tabBar options since that export will have many other properties but you will use only two of them here. Lets pass only those two reuqired to the tabBarOptions

Reply:
It is updated